src/test/rustdoc/unneeded-trait-implementations-title.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![crate_name = "foo"]
+
+pub struct Bar;
+
+// @!has foo/struct.Bar.html 'Trait implementations'
```

Comment:
Is the "implementations" being lowercase here going to cause a problem? I remember in the past we had a misspelled string in a `@!has` rule, which is always going to be false, but i don't remember whether the `htmldocck.py` check is case-sensitive.

Reply:
It is case sensitive (which caused me a few issues).

Reply:
It looks like the real header capitalizes the "I" there, so that should be changed.

Reply:
Oh indeed!